uptane-standard.md

```diff
@@ -481,7 +491,6 @@ The following information SHOULD be provided for each image on both the Image re
 
 The following information is CONDITIONALLY REQUIRED for each image on the Director repository IF that image is encrypted:
 
-* Information about filenames, hashes, and file size of the encrypted image
```

Comment:
Why was this removed?

Reply:
I removed it for clarity since it is already stated as a requirement in Section 5.2.3.1 "Metadata about Images".

Please let me know your feedback.

Reply:
Considering the ongoing conversation about primary's not having ability to decrypt all images, I have added this back.

Sorry for any confusion.